parsing/CSVBuilder.py

```diff
@@ -0,0 +1,68 @@
+import csv
+
+#This class will take a dictionary of strings and integers and transform it into a CSV file with the correct converted metric names.
+class CSVBuilder:
+    correctFormat = True
+    fname = ''
+    myDict = {}
```

Comment:
rename to _myDict to indicate that it is a "private" attribute.

(Nothing is actually private in Python, but most IDE's autocompleters will not offer it as a suggestion if it starts with an underscore or a double underscore.)